resources/assets/uad_lists.json

```
@@ -17633,7 +17633,7 @@
   {
     "id": "com.google.android.safetycenter.resources",
     "list": "Google",
-    "description": "Google safety center anti-virus spyware\n(https://safety.google). Disables the 'Please enable google play\nprotect' prompt when installing from F-Droid after disabling it in the\nPlay Store. Probably also disables gmail and browser malware notifications.\n\nUse a libre spam-blocking and DNS-blocking solution instead.",
+    "description": "Related to Google Safety Center (https://safety.google). Probably effects malware detection in new app installs, gmail, and chrome. Use a libre spam-blocking and DNS-blocking solution instead.",
```

```suggestion
    "description": "Related to Google Safety Center (https://safety.google). Probably affects malware detection in new app installs, Gmail, and Chrome. Alternatively, you can use a libre spam-blocking and DNS-blocking solution instead.",
```

Don't think you should indirectly force users to use something else. It's up to the user.